modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/compat/modal/Modal.es.js

```diff
@@ -108,9 +110,13 @@ class Modal extends Component {
 	syncOverlay(overlay) {
 		const willShowOverlay = overlay && this.visible;
 
-		dom[willShowOverlay ? 'enterDocument' : 'exitDocument'](
-			this.overlayElement
-		);
+		if (willShowOverlay) {
+			dom.enterDocument(this.overlayElement);
+
+			return;
```

Comment:
In this case I opted to return earlier for avoiding an `else` here

Reply:
it's okay... we'll remove `enterDocument` soon enough...